yarn-project/sequencer-client/src/publisher/sequencer-publisher.ts

```diff
@@ -721,9 +733,15 @@ export class SequencerPublisher {
         }
       },
     });
+    this.signaledPayloads.add(payload.toString());
```

Comment:
A few things to note here:
- it's totally reasonable for an attester to signal multiple times on the same payload. They can signal once per slot but they may be selected for multiple slots (even consecutive slots) and they should still be able to vote
- a publisher can be shared by multiple attesters